src/device.rs

```diff
@@ -799,6 +803,15 @@ impl Device {
         })
     }
 
+    pub fn receive_ping(&self) -> Result<String> {
```

Comment:
Wild idea: maybe we should expose/compile API methods, that are only used in tests, only in tests? I know that there was a libtelio space saving initiative :smile:

Reply:
Uniffi doesn't like conditional compilation like that unfortunately

Reply:
Not that wild, but good idea to not expose those methods in release. But will create separate issue for this if you don't mind.

Reply:
LLT-5451